tests/data/expression.diff

```diff
@@ -19,7 +19,7 @@
  (~int) and (not ((v1 ^ (123 + v2)) | True))
 -+really ** -confusing ** ~operator ** -precedence
 -flags & ~ select.EPOLLIN and waiters.write_task is not None
-++(really ** -(confusing ** ~(operator ** -precedence)))
+++(really**-(confusing**~(operator**-precedence)))
```

Comment:
Wait... what? You really think this is more readable in the new form? I think that once you hit a complex base or exponent, then there is no value in skipping the spaces.